lib/katello/engine.rb

```diff
@@ -258,6 +258,10 @@ def self.with_remote_execution?
     (RemoteExecutionFeature rescue false) ? true : false
   end
 
+  def self.with_katello_agent?
+    return true
```

Comment:
I'm assuming this is the part that will be updated later to pull the setting?

Reply:
Yeah once the feature from Eric is made, this will be changed to something better than just `true` or `false` lol. I marked the PR as blocked by another so it does not get merged on accident.

Reply:
I never look at the tags.  I would change it to a draft PR or add [WIP] to the beginning of the title. :stuck_out_tongue:

Reply:
Changed to [WIP] since Github won't allow me to change it to a draft